File: 58-LengthOfLastWord/58-LengthOfLastWord.java
// Last updated: 8/12/2026, 9:15:32 AM
class Solution {
    public int lengthOfLastWord(String s) {
        String str[] = s.split(" ");
        int n = str.length-1;
        int x = str[n].length();
        return x;    
    }
}

File: 58-LengthOfLastWord/58-LengthOfLastWord.cpp
// Last updated: 8/12/2026, 9:15:36 AM
class Solution {
public:
    int lengthOfLastWord(string s) {
        int end = s.length() - 1;

        while (end >= 0 && s[end] == ' ') {
            end--;
        }

        int start = end;
        while (start >= 0 && s[start] != ' ') {
            start--;
        }

        return end - start;        
    }
};